internal/workload/podspec_updates.go

```diff
@@ -669,6 +678,8 @@ func (s *updateState) addHealthCheck(p *cloudsqlapi.AuthProxyWorkload, c *corev1
 	// Add a port that is associated with the proxy, but not a specific db instance
 	s.addPort(port, proxyInstanceID{AuthProxyWorkload: types.NamespacedName{Namespace: p.Namespace, Name: p.Name}})
 	s.addProxyContainerEnvVar(p, "CSQL_PROXY_HTTP_PORT", fmt.Sprintf("%d", port))
+	s.addProxyContainerEnvVar(p, "CSQL_PROXY_ADMIN_PORT", fmt.Sprintf("%d", adminPort))
+	s.addProxyContainerEnvVar(p, "CSQL_PROXY_QUITQUITQUIT", "true")
```

Comment:
Do we want to turn this on by default? This is meant as a job-only endpoint.

Reply:
For now, I'll update this to only start the /quitquitquit endpoint when the customer explicitly sets AdminPort. 

Also, we need to give some though to the design for health checks, readiness, and shutdown. This is a much larger design effort than this one PR. Here's a reference: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/

Reply:
Aside from quitquitquit, we also have a debug flag in the proxy which uses the admin server. In the proxy, we only start the admin server if a caller has asked for quitquitquit or debug.

Reply:
I filed a bug for the readiness, see #219

Reply:
I think we need to reframe this PR as enabling quitquitquit, or enabling debug. The port is a detail of either (or both) of those.

Reply:
Good call. I've updated the PR.